pkg/datapath/loader/link.go

```diff
@@ -27,6 +27,14 @@ func SetXDPMode(mode string) error {
 			return fmt.Errorf("XDP Mode conflict: current mode is %s, trying to set conflicting %s",
 				option.Config.XDPMode, option.XDPModeLinkGeneric)
 		}
+	case option.XDPModeBestEffort:
```

Comment:
Could this case be combined with `case option.XDPModeNative` to avoid the code duplication?